#!/usr/bin/env python
PACKAGE = "hector_move_group_capabilities"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_distance_to_obstacle", double_t, 0, "Min raycast distance [m]", 0.05, 0.0, 100.0)
gen.add("max_distance_to_obstacle", double_t, 0, "Max raycast distance [m]", 3.2, 0, 100.0)
gen.add("secondary_rays_max_dist", double_t, 0, "Secondary rays max distance [m]",5.0, 0.0, 100.0)
gen.add("secondary_rays_opening_angle_deg", double_t, 0, "Secondary rays opening angle [deg]", 3.0, 0.0, 90.0)

exit(gen.generate(PACKAGE, "hector_move_group_capabilities", "OctomapRaycastCapability"))
